styleguide/_themes/derek/scss/patterns/cards/ias.scss

```diff
@@ -39,8 +39,13 @@
 .iasWidget {
   display: block;
   padding: 20px 10px 0;
+  text-align: center;
   text-decoration: none;
   width: $full-width;
+
+  p {
```

Comment:
@bighappyface Yeah this is to center the body whether or not there is a <p> tag around the text for IaS and Value Props. When using Legacy HTML there are no tags set but when using the Full HTML/WYSIWYG there will be a <p> tag assigned automagically.

Legacy HTML - /cloud/orchestration
Full HTML - /node/7366